eds4jinja2/builders/jinja_builder.py

```diff
@@ -41,7 +41,7 @@ def build_eds_environment(external_data_source_builders={**DATA_SOURCE_BUILDERS,
     return template_env
 
 
```

Comment:
non-blocking observation: We will have to decide what does `jinja_builder` and what does `report_builder` and eventually merge the two into a common functionality at the "service layer"